bigtable/reader.go

```diff
@@ -43,6 +43,7 @@ type ReadItem struct {
 	Row, Column string
 	Timestamp   Timestamp
 	Value       []byte
+	Labels      []string
```

Comment:
Looks like there are unit tests with ReadItem-- could you add unit tests there which exercise Labels as well?

Reply:
Added a test with a splitted cell and a complete cell cases